services/reddit/subreddit-subscribers.tester.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const { isMetric } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('subreddit-subscribers (valid subreddit)')
+  .get('/subreddit-subscribers/drums.json')
```

Comment:
You don't need the additional path reference here anymore, as the `createServiceTester` helper does a lot of the path prefixing for you
```suggestion
  .get('/drums.json')
```